Swift/Defaults.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  Defaults.swift
+//  CouchbaseLite
+//
+//  Copyright (c) 2022-present Couchbase, Inc All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+// THIS IS AN AUTOGENERATED FILE, MANUAL CHANGES SHOULD BE EXPECTED TO
+// BE OVERWRITTEN
+
+import Foundation
+
+public struct Defaults {
+	/// [false] Plaintext is not used, and instead binary encoding is used in log files
+	public static let defaultLogFileUsePlainText: Bool = kCBLDefaultLogFileUsePlainText.boolValue
```

Comment:
Same as objective-c. Group them into section. Alternatively I think extension can be used here as well.

Reply:
I think for consistency with Defaults files, it's better to keep in same Defaults struct itself, and group using `// MARK: group name`

```
public struct Defaults {

// MARK: LogFileConfiguration 

// MARK: FullTextIndexConfiguration

// MARK: ReplicatorConfiguration 

}
```

We could separate out into extension classes, sometime in future when it goes out of control?

Reply:
IMO, it doesn't need to be in sync with Objective-C. Which option do you prefer as a user? You can make decision.

Reply:
>  Which option do you prefer as a user?

My understanding, this constant will be used by very few users. Since it's the default value, so no need to set it. It will be useful for CBL devs, to set it. Am I right?

Personally, for me I am fine with both ways, 
* more cleaner/organized code seems like extensions, 
* we only have very limited number of constants and it is fit in one page itself(not feels like, worth spending formatting it may be?)

Reply:
I don't think it's about the number of constants. It's about API and API doc.

Reply:
> My understanding, this constant will be used by very few users. Since it's the default value, so no need to set it. It will be useful for CBL devs, to set it. Am I right?

They are served references for the default values. It's a user's choice to use it for setting the config values programmatically.

Reply:
If you think using Defaults is preferred because it's easier to use and organized enough, go for it. 

But if you think that using extension is better, but we only have 17 configs now so let don't spend more time on it. I feel that it's not a right argument. Changing the API later also means that the old API needs to deprecated.

Reply:
ok. I think, I will agree and go for extension.

I was thinking from a stand point where we are keeping the constants in a single place and exposed outside just for reference. Not given enough thought from an API / API doc stand point. I get your point now.

Reply:
Updated the new format as comment: https://issues.couchbase.com/browse/CBL-3744